packages/cmf/src/componentState.js

```diff
@@ -33,18 +38,33 @@ export function getStateAccessors(dispatch, name, id, DEFAULT_STATE) {
 			if (typeof state === 'function') {
 				dispatch(applyCallback(state, name, id));
 			} else {
-				dispatch(actions.componentsActions.mergeComponentState(name, id, state));
+				const componentState = actions.componentsActions.mergeComponentState(name, id, state);
+				dispatch({
+					id,
+					type: `${name}.setState`,
+					cmf: { componentState },
+				});
```

Comment:
Can you externalize this dispatch in a `dispatchComponentState()` to write this payload wrapper in only 1 place instead of 3-4 ?